Introduce and structure label settings and text formats page (#4129)
qgis/QGIS-Documentation: source/docs/user_manual/style_library/label_settings.rst

   .. contents::
      :local:

> Labels are textual information you can display on vector features. They add

Reviewer: ```suggestion
Labels are textual information you can display with vector features. They add
```

Reviewer: ?

> details you could not necessarily represent using symbols.

Reviewer: ```suggestion
information that is difficult to represent using symbols.
```

> The :guilabel:`Style Manager` dialog allows you to create a set of labels or

Reviewer: ```suggestion
The :guilabel:`Style Manager` dialog allows you to create a set of formatting styles for labels
```

> text formats (ie the appearance of the text, including font, size, colors,

Reviewer: ```suggestion
(ie the appearance of the text, including font, size, colors,
```

Author: This is not really the message I wanted to convey, ie from this dialog you create label settings AND text formats; 2 different items despite the latter is included in the former (which complicates description). the idea was also to introduce the "text formats" expression before it gets its own section.

Reviewer: OK. I was probably confused by the use of "or" (in "labels or text formats").

shadow, background...). Each of these items could later be applied to layers in
the |labeling| :guilabel:`Labels` tab of the vector :guilabel:`Layer Properties`
dialog or :guilabel:`Layer Styling` panel or using the |labeling| :sup:`Layer
Labeling Options` button of the **Labels toolbar**. You can also directly
configure them in the abovementioned dialogs.

.. It could be nice to briefly expose the "label settings" tab of the style
 manager, with some sample of label settings in it.

> The :guilabel:`Label Settings` dialog provides you with all the needed and

Reviewer: ```suggestion
The :guilabel:`Label Settings` dialog allows you to
```

> appropriate capabilities to configure smart labeling on vector layers. Setting

Reviewer: ```suggestion
to configure smart labeling for vector layers. Setting
```

a label includes configuring the :ref:`text format <text_format>`, and how
the label relates with the features or other labels (through :ref:`placement
<labels_placement>`, :ref:`rendering <labels_rendering>` and callout).

.. _text_format:

> Formatting the label's text

Reviewer: ```suggestion
Formatting the label text
```

> ===========================

Reviewer: ```suggestion
=========================
```
